crates/nu-command/src/filesystem/mv.rs

```diff
@@ -50,7 +50,7 @@ impl Command for Mv {
         let interactive = call.has_flag("interactive");
         let force = call.has_flag("force");
 
-        let path: PathBuf = current_dir()?;
+        let path = PathBuf::from(current_dir(engine_state, stack)?);
```

Comment:
one small change we might want to do is a helper method on `stack` that gets the current_dir as a PathBuf for you

Reply:
@jntrnr It would be nice, however, the `current_dir()` has a secret engine dependency: In case PWD env var is not a string, we need `eval_block()` to convert it to string and I didn't want to introduce this dependency to the protocol. We could, of course, require that PWD must always be a string. Then, we could move it to the stack. (I'd prefer having `current_dir()` return a PathBuf and `current_dir_str()`return a String since both are used frequently.)